utils/src/main/java/software/amazon/awssdk/utils/internal/proxy/ProxySystemPropertyConfigProvider.java

```diff
@@ -46,7 +46,7 @@ private static Integer safelyParseInt(String string) {
         try {
             return Integer.parseInt(string);
         } catch (Exception e) {
-            log.error(() -> "Failed to parse string" + string, e);
+            log.warn(() -> "Failed to parse string. The failure will be ignored." + string, e);
```

Comment:
We can just log 
```java
log.warn(() -> "Failed to parse string.", e);
```
The warn exception  message already prints the String as below
```
15:45:50.229 [main] WARN  software.amazon.awssdk.utils.internal.proxy.ProxySystemPropertyConfigProvider - Failed to parse string.
java.lang.NumberFormatException: For input string: ""
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:592)
	at java.lang.Integer.parseInt(Integer.java:615)
	at software.amazon.awssdk.utils.internal.proxy.ProxySystemPropertyConfigProvider.safelyParseInt(ProxySystemPropertyConfigProvider.java:47)
	at java.util.Optional.map(Optional.java:215)
	at software.amazon.awssdk.utils.internal.proxy.ProxySystemPropertyConfigProvider.port(ProxySystemPropertyConfigProvider.java:58)
	at software.amazon.awssdk.transfer.s3.StandaloneSnippet.main(StandaloneSnippet.java:21)
```

Reply:
+1

Reply:
Done.